### PR TITLE
[release] V6.18.6

### PR DESCRIPTION
**Pull request by romgrk**
### Release the packages

- [ ] Checkout the last version of the working branch
- [ ] Make sure you have the latest dependencies installed: `yarn`.
- [ ] Build the packages: `yarn release:build`.
- [ ] Release the versions on npm: `yarn release:publish` (you need your 2FA device).
- [ ] Push the newly created tag: `yarn release:tag`.

### Publish the documentation

The documentation must be updated on the `docs-vX` branch (`docs-v4` for `v4.X` releases, `docs-v5` for `v5.X` releases, ...)

- [ ] Push the working branch on the documentation release branch to deploy the documentation with the latest changes.

```bash
git push upstream next:docs-next -f
```

You can follow the deployment process [on the Netlify Dashboard](https://app.netlify.com/sites/material-ui-x/deploys?filter=docs-next)
Once deployed, it will be accessible at https://material-ui-x.netlify.app/ for the `docs-next` deployment.

### Publish GitHub release

- [ ] After documentation is deployed, publish a new release on [GitHub releases page](https://github.com/mui/mui-x/releases)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,60 @@
...
+- 🐞 Bugfixes
+
+
+From https://github.com/mui/mui-x/pull/11490
+
+- [charts] Allow percentage values for pie chart center and radius (#11464) @alexfauquette
+- [charts] Make error message more explicit (#11457) @alexfauquette
+- [charts] Make the helper `ChartsText` component public (#11370) @alexfauquette
+- [charts] Improve dataset typing (#11372) @alexfauquette
+- [charts] Fix size overflow (#11385) @alexfauquette
```
Comment (cherniavskii) [inline CHANGELOG.md:22]: This should be moved to the charts section

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,60 @@
...
+
+### Data Grid
+
+#### `@mui/x-data-grid@6.18.6`
+
+- [DataGrid] Fix typos in the JSDoc (@flaviendelangle) (#11475) @github-actions[bot]
```
Comment (cherniavskii) [inline CHANGELOG.md:28]:
```suggestion
- [DataGrid] Fix typos in the JSDoc (#11475) @flaviendelangle
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,60 @@
...
+
+#### `@mui/x-data-grid-premium@6.18.6` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')
+
+Same changes as in `@mui/x-data-grid-pro@6.18.6`.
+
+### Date Pickers
+
+#### `@mui/x-date-pickers@6.18.6`
+
+- [fields] Fix section pasting (#11467) @github-actions[bot]
```
Comment (cherniavskii) [inline CHANGELOG.md:43]:
```suggestion
- [fields] Fix section pasting (#11467) @LukasTy
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,60 @@
...
+### Charts / `@mui/x-charts@6.18.6`
+
+- [charts] Cherry pick charts updates (changelog in the description) (#11490) @alexfauquette
+
+### Docs
+
+- [docs] Document false default values for boolean props (#11489) @cherniavskii
+- [docs] Improve Pickers `name` prop examples (#11442) @LukasTy
+- [docs] Limit `date-fns` package to v2 in codesandbox (@LukasTy) (#11478) @github-actions[bot]
+- [test] Reload the page if its blank and there are no links to the remaining tests (#11471) @github-actions[bot]
```
Comment (cherniavskii) [inline CHANGELOG.md:58]:
```suggestion
- [docs] Limit `date-fns` package to v2 in codesandbox (#11478) @LukasTy
- [test] Reload the page if its blank and there are no links to the remaining tests (#11471) @cherniavskii
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,56 @@
...
+
+- [fields] Fix section pasting (#11467) @LukasTy
+
+#### `@mui/x-date-pickers-pro@6.18.6` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-date-pickers@6.18.6`.
+
+### Charts / `@mui/x-charts@6.18.6`
+
+- [charts] Cherry pick charts updates (changelog in the description) (#11490) @alexfauquette
```
Comment (alexfauquette) [inline CHANGELOG.md:42]:
```suggestion
```

Comment (alexfauquette) [reply CHANGELOG.md:42]: This is the title of the PR that added a batch of cherry-pick. what matter is the other ones

**Review by cherniavskii — APPROVED** [review]

File: packages/x-charts/package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-charts",
-  "version": "6.18.3",
+  "version": "6.18.4",
```
Comment (LukasTy) [inline packages/x-charts/package.json:3]: Nitpick: We usually sync the released version with the current root release version. 😉

**Pull request merged**